test/functional/wallet_resendwallettransactions.py

```diff
@@ -39,9 +39,9 @@ def run_test(self):
         self.log.info("Create a new transaction and wait until it's broadcast")
         txid = int(node.sendtoaddress(node.getnewaddress(), 1), 16)
 
-        # The rebroadcast scheduler can is first called 1 sec after startupp
-        # (see fNextResend in ResendWalletTransactions()). Sleep for just over
-        # a second to be certain that it's been called.
+        # Wallet rebroadcast is first scheduled 1 sec after startup (see
```

Comment:
In commit "[wallet] Remove unnecessary Chain::Lock parameter from ResendWalletTransactions" (4aa062760e5b6d91e3528833eff81d6a6cf96ff5)

Comment change should go in the previous commit which adds the comment and the sleep: "[wallet] Schedule tx rebroadcasts in wallet." (b1d1c414889aa2573806fe2d925651be66335bc4), since this isn't related to removing chain_lock.